beacon-chain/sync/subscriber_data_column_sidecar.go

```diff
@@ -29,6 +31,12 @@ func (s *Service) dataColumnSubscriber(ctx context.Context, msg proto.Message) e
 		return errors.Wrap(err, "reconstruct data columns")
 	}
 
+	// Trigger getBlobsV2 when receiving data column sidecar
+	if err := s.triggerGetBlobsV2ForDataColumnSidecar(ctx, root); err != nil {
+		// Log error but don't fail - this is a best-effort retry trigger
```

Comment:
Don't really agree. Please wrap and return the error and let the caller to decide.